hardware/Meteorologisk.cpp

```diff
@@ -48,22 +48,90 @@ std::string ReadFile(std::string filename)
 }
 #endif
 
+#define Meteorologisk_URL "https://api.met.no/weatherapi/locationforecast/2.0/complete.json"
+#define Meteorologisk_forecast_URL "https://darksky.net/#/f/"
+
 CMeteorologisk::CMeteorologisk(const int ID, const std::string &Location) :
 	m_Location(Location)
 {
 	m_HwdID=ID;
-	Init();
+	m_URL = "";
```

Comment:
Thank you for the PR.
You don't have to initialize a std::string in a constructor of a class. This is already done

Reply:
done